homeassistant/components/switchbot_cloud/__init__.py

```diff
@@ -153,14 +178,24 @@ async def make_device_data(
         )
         devices_data.vacuums.append((device, coordinator))
 
-    if isinstance(device, Device) and device.device_type.startswith("Smart Lock"):
+    if isinstance(device, Device) and device.device_type in [
```

Comment:
Why changing startswith ?

Reply:
My app has device it called [Lock Ultra], in init.py, ->func->make_device_data will add all Smart Lock
but in sensor.py, there is no description for it ,it raise a error ,it not friendly for new device,

Reply:
https://github.com/home-assistant/core/issues/146768#issuecomment-2976064400

Reply:
it can temporarily solve the lock ultra import error, if you don't want to  fixed it here , I   can make it back